src/main/java/aquality/selenium/browser/devtools/DevToolsFactory.java

```diff
@@ -0,0 +1,19 @@
+package aquality.selenium.browser.devtools;
+
+import aquality.selenium.configuration.IBrowserProfile;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.devtools.DevTools;
+import org.openqa.selenium.devtools.HasDevTools;
+
+public class DevToolsFactory {
+
+    public static DevTools getDevTools(WebDriver driver, IBrowserProfile browserProfile) {
+        DevTools tools;
+        try {
+            tools = ((HasDevTools)driver).getDevTools();
```

Comment:
I'm not sure why do we need a static DevToolsFactory class. There is only one method in here, more like a factory method rather then an abstract factory. Could we move this method to BrowserDevTools class please?

Reply:
Also I believe we may want to use Augmenter here for the case of RemoteWebDriver:

    driver = new Augmenter().augment(driver);
    DevTools devTools = ((HasDevTools) driver).getDevTools();

Like they do here: https://www.selenium.dev/documentation/webdriver/bidirectional/chrome_devtools/#emulate-geo-location-with-the-remote-webdriver